plugins/saveactions.c

```diff
@@ -393,19 +689,54 @@ static gboolean auto_save(gpointer data)
 
 static void autosave_set_timeout(void)
 {
+	if (autosave_src_id != 0)
```

Comment:
this fixes a bug when disabling `autosave` function didn't actually stop autosave timer
Unrelated to new functionality